frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -466,8 +493,9 @@ export function CandidateContestChoices({
   targetMarkPosition,
 }: CandidateContestChoicesProps): JSX.Element {
   const { t } = useTranslation();
-  const writeInCandidates = vote?.filter((c) => c.isWriteIn);
-  const remainingChoices = [...Array.from({ length: contest.seats }).keys()];
+  const writeInItemKeys = [...Array.from({ length: contest.seats }).keys()].map(
```

Comment:
Possibly easier to understand?
```suggestion
  const writeInItemKeys = take(contest.seats, integers()).map(
```